blockdag/dag.go

```diff
@@ -605,20 +605,13 @@ func (dag *BlockDAG) connectBlock(node *blockNode,
 	return chainUpdates, nil
 }
 
-// calcMultiset returns the multiset of the UTXO of the given block with the given transactions.
-func (node *blockNode) calcMultiset(dag *BlockDAG, transactions []*util.Tx, acceptanceData MultiBlockTxsAcceptanceData, selectedParentUTXO, pastUTXO UTXOSet) (*secp256k1.MultiSet, error) {
+// calcMultiset returns the multiset of the UTXO of the given block.
+func (node *blockNode) calcMultiset(dag *BlockDAG, acceptanceData MultiBlockTxsAcceptanceData, selectedParentUTXO UTXOSet) (*secp256k1.MultiSet, error) {
```

Comment:
It seems like calcMultiset becomes redundant if it just calls pastUTXOMultiSet.

Reply:
Maybe so, but I'd still rather have this function. How a multiset is being calculated is implementation details within calcMultiset.

Reply:
Alright, but in that case I guess it can simply be be 
```
func calcMultiset(lalalala){
    return node.pastUTXOMultiSet(dag, acceptanceData, selectedParentUTXO)`
}
```